bokeh/models/tools.py

```diff
@@ -186,11 +186,12 @@ class TapTool(Tool):
 
 @abstract
 class InspectTool(Tool):
+    pass
 
-    active = Bool(True, help="""
-    Whether the tool is intially active or not. If set to ``False``, the user
-    will have to click tool's button to active it.
-    """)
+    #active = Bool(True, help="""
+    #Whether the tool is intially active or not. If set to ``False``, the user
+    #will have to click tool's button to active it.
+    #""")
```

Comment:
My only comment is what about Button Actions (e.g. preview save tool)? they are just there, to be clicked, not active or not active .